bootstrap.py

```diff
@@ -1,16 +1,11 @@
-##############################################################################
+# Per the Zope Public License Version 2.0, you must update this file-specific
```

Comment:
we only redistribute this file - this is standardized, so we should not modify it.